source/_integrations/devolo_home_control.markdown

```diff
@@ -113,6 +113,11 @@ The integration provides support for the following Z-Wave devices:
 
 - devolo Siren
 
+## Known limitations
+
+Device addition and removal are not supported by this integration as the local API rejects these operations.
+To add or remove devices, please use the devolo Home Control app or the web interface.
```

Comment:
I don't think this is an limitation, more of an instruction on how to remove devices?

../Frenck

Reply:
My main motivation was to prevent some to think one can replace the devolo world completely with Home Assistant, especially under the aspect, that devolo will discontinue the app and the web interface end of this year. 

Would you just remove the last sentence like in "the user probably knows what to do" or would you skip the complete section like in "Home Assistant users typically know how that hub integrations need other ways to remove devices"? I don't use a hub integration in my productive setup so I don't have other references.